packages/versioning/src/lib.rs

```diff
@@ -94,18 +94,11 @@ pub fn initialize(storage: &mut dyn Storage, version: Version) -> StdResult<()>
     VERSION_STORAGE_KEY.save(storage, &version)
 }
 
-pub fn update_software<ContractError>(
-    storage: &mut dyn Storage,
-    new: Version,
-) -> Result<ReleaseLabel, ContractError>
-where
-    StdError: Into<ContractError>,
-{
+pub fn update_software(storage: &mut dyn Storage, new: Version) -> Result<ReleaseLabel, StdError> {
```

Comment:
just noticed that the return error is not consistent with `update_software_and_storage`. Update it as well